packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.scss

```diff
@@ -1,5 +1,7 @@
 .product-payment {
   .subscription-ready {
+    margin-top: -24px;
```

Comment:
As usual, my CSS is terrible. Suggestions totally welcome

Reply:
Sorry, should have started a review, looking at this now

Reply:
I think we actually want the extra space here. @ryanfeeley confirmed in another issue that he liked 70px of space between the logo and content before.

70px of space
<img width="722" alt="image" src="https://user-images.githubusercontent.com/13018240/69452680-dc5a1c00-0d27-11ea-9e0b-d31c0658a51e.png">

vs subtracting 24px
<img width="705" alt="image" src="https://user-images.githubusercontent.com/13018240/69452701-e8de7480-0d27-11ea-895f-d7b8ac593b21.png">


If we want to subtract the 24px here, it's cleaner to adjust the parent padding based on this - cleanest to set it to `46px` and then increase it where we want it increased, or another extra class on here to change the `padding-top`... not worth the effort for this. If we want to leave the negative margin-top, let's add a `// TO DO` here for hackiness and we can address it during the settings redesign.

Reply:
@LZoog great assumption, I think the top one with more space reads more easily. I know that @pwalm is working with branding on a grid, so that should take some of the guesswork out of this in future.

Reply:
Sweet! Less hackiness 🙌

Reply:
Pushed an update to remove this.